examples/json-access/json_access.bal

```diff
@@ -0,0 +1,52 @@
+import ballerina/io;
+
+public function main() {
+    // Define a variable of type `json` that holds a mapping value.
+    json j1 = {
+        fname: "Mary",
+        lname: "Jane",
+        address: {
+            line: "20 Palm Grove",
+            city: "Colombo 03",
+            country: "Sri Lanka"
+        }
+    };
+
+    // Field access is allowed on the `json` typed variable, but the return
+    // type would be a union of `json` and `error`. An error will be returned
+    // either if the value on which the access is done is not a mapping or if
+    // it is a mapping, but it does not contain the particular key.
```

Comment:
it is mapping. However, it does not contain the particular key.

**Not clear why mapping is used as a noun here**

Reply:
This is pretty much from the spec. https://github.com/ballerina-platform/ballerina-spec/blob/master/lang/lib/value.bal#L85

We've used mappings to represent structural values that map a key to a value - maps, records.